* Fixed and error which was causing incorrect sorting for files with creation dates far in the past
===================================================================
--- timespec.h	2005-09-21 23:05:39.000000000 -0700
+++ timespec-fixed.h	2017-04-28 21:23:21.125585452 -0700
@@ -45,8 +45,12 @@
 static inline int
 timespec_cmp (struct timespec a, struct timespec b)
 {
-  int diff = a.tv_sec - b.tv_sec;
-  return diff ? diff : a.tv_nsec - b.tv_nsec;
+  if (a>b)
+	return 1;
+  if (b>a)
+	return -1;
+  if (a==b)
+	return 0;
 }
 
 # if ! HAVE_DECL_NANOSLEEP
